src/AppBundle/Resources/config/routing.yml

```diff
@@ -1,30 +1,11 @@
-
 app_add_component:
-    path:     /components/add
-    defaults: { _controller: AppBundle:App:addComponent }
+    path:     /components/{dashboardId}/add
+    defaults: { _controller: AppBundle:Component:addComponent }
+    requirements:
+        componentId: \d+
```

Comment:
I need to know the dashboard to associate the component and generate the link to return on it